pull/github.go

```diff
@@ -933,3 +940,70 @@ func isNotFound(err error) bool {
 	}
 	return false
 }
+
+type SignatureType string
+
+const (
+	SignatureGpg     SignatureType = "GpgSignature"
+	SignatureSmime   SignatureType = "SmimeSignature"
+	SignatureUnknown SignatureType = "UnknownSignature"
```

Comment:
Is this type used anywhere?

Reply:
No, I thought it might be necessary at one point but forgot to remove it. Done